### PR TITLE
Made the new tab button touch screen friendly

### LINKED ISSUE #5925: New tab button is not touch screen friendly
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
If using a touch screen or when on Windows (which is pickier about right click), the new tab button is much harder to do a long press on.  After investigating, it turns out the hitbox is only for the actual SVG itself

**Expected behavior:**
Clicking should be expanded to the size of the parent element or the height of the tabs area

- Platform (Win7, 8, 10? macOS? Linux distro?):
all

- Brave Version:
0.12.10

- Steps to reproduce:
    1. Right click all around the new tab button
    2. Notice that the new tab menu doesn't show up very often, because the hitbox is hard to click

- Screenshot if needed:
Here's a screenshot on Windows. Notice you can make the system menu show up even when you right click within the bounds of the +. It only seems to show if you DIRECTLY right click / long press on the +
screen shot 2016-11-30 at 1 01 59 am

- Any related issues:

### PR DESCRIPTION
**Pull request by luixxiul**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Fixes #5925

Auditors: @bsclifton

Test Plan: make sure clicking is expanded enough

### CODE REVIEW

**Comment by luixxiul** [comment]
work in progress :-)

**Comment by luixxiul** [comment]
ok, ready for review.

File: less/tabs.less [hunk 1]
```diff
@@ -36,22 +36,35 @@
...
+
+  .newFrameButton {
+    background: @buttonColor;
+    width: 14px;
+    height: 24px;
+    line-height: 24px;
+    -webkit-mask-image: url('../img/toolbar/newtab_btn.svg');
+    -webkit-mask-repeat: no-repeat;
+    -webkit-mask-position: center;
+    -webkit-mask-size: 12px 12px;
```
Comment (luixxiul) [inline less/tabs.less]: the plus image is 12x12 and centered, increasing the height of the button (which is clickable) to 24px.

**Comment by bsclifton** [comment]
@jkup would you be able to review this?

**Comment by jkup** [comment]
Yep I'll review now.

**Comment by bsclifton** [comment]
Awesome! I'll try the Windows version...

**Comment by jkup** [comment]
++ from me

This approach is cool, I didn't know about it!

Although I'm not really able to repro the initial issue on my Mac so hopefully @bsclifton can merge if the original issue is fixed on Windows!

**Comment by bsclifton** [comment]
Looks great :smile: Awesome job, @luixxiul!

Screenshot of the hitbox, so everybody can see. It now extends all the way up and down
image

**Comment by luixxiul** [comment]
Labelled `QA/no qa needed` as this PR was superseded with #6474

**Pull request merged**